package.json

```diff
@@ -8,27 +8,27 @@
   },
   "main": "polyfill/lib/index.mjs",
   "dependencies": {
-    "ecmarkup": "^3.22.0",
```

Comment:
The split between dependencies and devDependencies is a bit weird for the root package anyway. The package that's published to npm is only the `polyfill/` directory, and for the rest (spec, documentation, meeting minutes) there really are no runtime dependencies as such.